bootstrap.sh

```diff
@@ -28,12 +28,7 @@ function setup_mason() {
     (cd ./.mason && git fetch > /dev/null 2>&1 && git checkout ${MASON_VERSION} > /dev/null 2>&1)
   fi
 
-  wd=$(pwd)
-  export MASON_DIR=${wd}/.mason
-  export MASON_HOME=${wd}/mason_packages/.link
-  export PATH=${wd}/.mason:$PATH
-  export CXX=${CXX:-clang++}
-  export CC=${CC:-clang}
+  export MASON_DIR=$(pwd)/.mason
```

Comment:
Note: we'll be able to remove this line once upgraded to latest mason: https://github.com/mapbox/mason/pull/151